### PR TITLE
examples(latency): Add example to calculate client and server latency.

### PR DESCRIPTION
**Pull request by danielmai**
This is for CLOUD-260.

This adds an example to get the client-side and the server-side latency of a query.

The important bit is the following snippet:

https://github.com/dgraph-io/dgraph-js-http/blob/88ffe9400069489b18526090b35ddf9f55aff125/examples/latency/index-async-await.js#L112-L115

**Client-side latency** is calculated using `console.time` and `console.timeEnd` around a query call to capture the client-side latency.

**Server-side latency** is calculated by Dgraph itself. This example prints it out from the query response `res.extensions.server_latency`.

---

Example command and output:
<pre>
$ node index-async-await.js --addr https://$MY_DGRAPH_CLOUD_ENDPOINT --api-key $MY_API_KEY --drop-all
Created person named "Alice" with uid = 0xbd

All created nodes (map from blank node names to uids):
blank-0 => 0xbd
dg.3043386991.125 => 0xbe
dg.3043386991.126 => 0xbf
dg.3043386991.127 => 0xc0

Query: query all($a: string) { all(func: eq(name, $a)) { uid name age married loc dob friend { name age } school { name } } } Vars: { '$a': 'Alice' }
<strong>Query client latency: 39.371ms</strong>
<strong>Query server latency: {"parsing_ns":299313,"processing_ns":1606551,"encoding_ns":361067,"assign_timestamp_ns":1389885,"total_ns":3770262}</strong>
Number of people named "Alice": 1
{
  uid: '0xbd',
  name: 'Alice',
  age: 26,
  married: true,
  loc: { type: 'Point', coordinates: [ 1.1, 2 ] },
  dob: '1980-02-02T07:00:00Z',
  friend: [ { name: 'Bob', age: 24 }, { name: 'Charlie', age: 29 } ],
  school: [ { name: 'Crown Public School' } ]
}

DONE!
</pre>

The following flags are available in this example:
* `--addr`: The Dgraph address. Can be a Dgraph Cloud endpoint following the instructions: https://github.com/dgraph-io/dgraph-js-http#create-a-client-for-dgraph-cloud-endpoint. Defaults to `http://localhost:8080`.
* `--api-key`: The Dgraph Cloud API Key. Required if the `--addr` is a Dgraph Cloud endpoint.
* `--drop-all`: Call DropAll on Dgraph before loading the data for this example.

This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph-js-http/50)

### CODE REVIEW

File: examples/latency/index-async-await.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        Object.keys(assigned.data.uids).forEach((key) => console.log(`${key} => ${assigned.data.uids[key]}`));
+        console.log();
+    } finally {
+        // Clean up. Calling this after txn.commit() is a no-op
+        // and hence safe.
+        await txn.discard();
+    }
+}
+
+// Query for data.
```
Comment (anand-chandrashekar) [inline examples/latency/index-async-await.js:90]: We can add some documentation for this method. "This method also measures the net time taken on the client side to query data."

File: examples/latency/index-async-await.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            school {
+                name
+            }
+        }
+    }`;
+    const vars = { $a: "Alice" };
+
+    console.time('Query client latency');
+    const res = await dgraphClient.newTxn().queryWithVars(query, vars);
+    console.timeEnd('Query client latency');
```
Comment (anand-chandrashekar) [inline examples/latency/index-async-await.js:116]: Suggestion: we can consider printing the query on console here along with the latency.

**Review by anand-chandrashekar — APPROVED** [review]

File: examples/latency/index-async-await.js (hunk 1 above)
Comment (danielmai) [reply examples/latency/index-async-await.js:90]: I've added a doc comment.

File: examples/latency/index-async-await.js (hunk 2 above)
Comment (danielmai) [reply examples/latency/index-async-await.js:116]: Good point. I've added a log for the query (it collapses the query to a single line):

<pre>
<strong>Query: query all($a: string) { all(func: eq(name, $a)) { uid name age married loc dob friend { name age } school { name } } } Vars: { '$a': 'Alice' }</strong>
Query client latency: 35.8ms
Query server latency: {"parsing_ns":224023,"processing_ns":1479858,"encoding_ns":272385,"assign_timestamp_ns":1285762,"total_ns":3359570}
Number of people named "Alice": 1
</pre>

**Pull request merged**